doc/source/indexing.rst

```diff
@@ -390,6 +395,33 @@ For getting a value explicitly (equiv to deprecated ``df.get_value('a','A')``)
    # this is also equivalent to ``df1.at['a','A']``
    df1.loc['a', 'A']
 
+.. _indexing.slicing_with_labels:
+
+Slicing with labels
+~~~~~~~~~~~~~~~~~~~
+
+When using ``.loc`` with slices, if both the start and the stop labels are
+present in the index, then elements *located* between the two (including them)
+are returned:
+
+.. ipython:: python
+
+   s = pd.Series(np.random.random(5), index=[0,3,2,5,4])
```

Comment:
can you do with with 

```
s = pd.Series(list('abcde'), index=[0, 3, 2, 5, 4])
```

instead. I think a bit more clear on what is getting selected.